lua/entities/gmod_wire_expression2/core/custom/prop.lua

```diff
@@ -720,9 +720,12 @@ e2function table entity:ragdollGetPose()
 
 		pose.n[k] = value
 		pose.ntypes[k] = "t"
+		size = size + 1
 	end
 
-	pose.size = #pose.n
+	pose.stypes._origina = "a"
+	pose.s._origina = bones[0]:GetAngles()
```

Comment:
Do E2 arrays support 0 index?

Reply:
They can use them but they won't work on `foreach`. This is a table anyway. Numerical indices are already being used for the bones themselves.

Reply:
> Also should _origina be documented

It's kind of an internal thing but I guess I could. Somewhere.